tools/walletextension/main/cli.go

```diff
@@ -32,6 +32,10 @@ const (
 	logPathName    = "logPath"
 	logPathDefault = "wallet_extension_logs.txt"
 	logPathUsage   = "The path to use for the wallet extension's log file"
+
+	persistencePathName    = "persistencePath"
+	persistencePathDefault = ""
```

Comment:
the value does not match the documented default

Reply:
The code at the moment has a conditional for an empty string for the persistencePathName to then be interpreted as the ~/.obscuro/wallet_extension_persistence ... see walletextension/persistence/persistence.go;

```
func NewPersistence(hostAddr string, persistenceFilePath string) *Persistence {
	// Sets up the persistence file and returns its path. Defaults to the user's home directory if the path is empty.
	if persistenceFilePath == "" {
		homeDir, err := os.UserHomeDir()
		if err != nil {
			panic("cannot create persistence file as user's home directory is not defined")
		}
		obscuroDir := filepath.Join(homeDir, obscuroDirName)
		err = os.MkdirAll(obscuroDir, 0o777)
		if err != nil {
			panic(fmt.Sprintf("could not create %s directory in user's home directory", obscuroDirName))
		}

		persistenceFilePath = filepath.Join(obscuroDir, persistenceFileName)
	}
```

keeping this as an empty string was minimal code change.

Reply:
Note this is not exposed to the user i.e. the help message is as below;

```
Morays-MacBook-Pro:artifacts moraygrieve$ ./wallet_extension -h
Usage of ./wallet_extension:
  -logPath string
    	The path to use for the wallet extension's log file (default "wallet_extension_logs.txt")
  -nodeHost testnet.obscu.ro
    	The host on which to connect to the Obscuro node. Default: testnet.obscu.ro. (default "testnet.obscu.ro")
  -nodePortHTTP int
    	The port on which to connect to the Obscuro node via RPC over HTTP. Default: 13000. (default 13000)
  -nodePortWS int
    	The port on which to connect to the Obscuro node via RPC over websockets. Default: 13001. (default 13001)
  -persistencePath string
    	The path to use for the wallet extension's persistence file. Default: ~/.obscuro/wallet_extension_persistenc
```e